.github/workflows/dockerfile-validation.yml

```diff
@@ -0,0 +1,20 @@
+name: Dockerfiles linting
```

Comment:
Validation of `Dockerfile` files. This is relying on [this very neat tool](https://github.com/replicatedhq/dockerfilelint). We can further improve this by passing custom options to the linting tool, but this already helps quite a bit as it is.